requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy < 1.22
 xarray
-dask[array] >= 2021.10.0
-distributed >= 2021.10.0
+dask[array] >= 2022.01.0
+distributed >= 2022.01.0
```

Comment:
This needs changing in setup.cfg too. (I often forget to do that.)

Reply:
Perhaps we should add a `Dependencies` section to [Contribute to sgkit](https://pystatgen.github.io/sgkit/latest/contributing.html#contributing-to-code) that mentions this detail and any other tricky bits you can think about? Python dependency management is terrifying to me and any wisdom you can share would be much appreciated!

Reply:
Ah, thanks

Reply:
> Perhaps we should add a `Dependencies` section to [Contribute to sgkit](https://pystatgen.github.io/sgkit/latest/contributing.html#contributing-to-code) that mentions this detail and any other tricky bits you can think about?

Good idea. Opened #847